pkg/common/utils.go

```diff
@@ -81,3 +81,20 @@ func GetAccountNumFromAWSID(AWSID string) (string, error) {
 	accountNum := (z & mask) >> 7
 	return fmt.Sprintf("%012d", accountNum), nil
 }
+
+// SliceContainsString searches a slice to determine if it contains a specified string.
+func SliceContainsString(origTargetString string, stringSlice []string, ignoreCase bool) (bool, string) {
+	targetString := origTargetString
+	if ignoreCase {
+		targetString = strings.ToLower(origTargetString)
+	}
+	for _, stringFromSlice := range stringSlice {
+		if ignoreCase {
```

Comment:
suggestion: If `ignoreCase` is true, we can avoid the overhead of calling `strings.ToLower` on each slice item by performing the conversion once.

Ex:
```
if ignoreCase {
		targetString := strings.ToLower(origTargetString)
		for _, stringFromSlice := range stringSlice {
			if targetString == strings.ToLower(stringFromSlice) {
				return true, origTargetString
			}
		}
	} else {
		for _, stringFromSlice := range stringSlice {
			if origTargetString == stringFromSlice {
				return true, origTargetString
			}
		}
	}
	return false, ""
```

Reply:
It currently only does the `ToLower` conversion if `ignoreCase` is true. The original way does the check on every iteration, uses the same loop.

Reply:
Oh right duh!